AppLovinMAXDemo/Plugins/AppLovinMAX/AppLovinMAX.uplugin

```diff
@@ -0,0 +1,24 @@
+{
+	"FileVersion": 3,
+	"Version": 1,
+	"VersionName": "1.0",
+	"FriendlyName": "AppLovin MAX",
+	"Description": "AppLovin MAX Unreal Engine 4 Plugin for Android and iOS",
+	"Category": "Advertising",
+	"CreatedBy": "AppLovin Corporation",
+	"CreatedByURL": "https://www.applovin.com",
+	"DocsURL": "",
```

Comment:
Question - what's this file for anyways? Also this will be the docs URL:
```suggestion
	"DocsURL": "https://dash.applovin.com/documentation/mediation/unreal/getting-started/integration",
```

Reply:
It's essentially the `package.json` for Unreal plugins, containing all the basic plugin info. Unreal looks for this file to actually find the plugin, plus shows this info in the plugins list.